modules/imaging_browser/README.md

```diff
@@ -5,6 +5,9 @@
 The imaging browser is intended to allow users to view candidate
 scan sessions collected for a study.
 
+It is used primarily by imaging specialists to do online QC of
+imaging data that has been inserted into LORIS.
+
```

Comment:
@driusan @xlecours 
I would say that besides imaging specialists doing online QC, it is also used by the project radiologist to perform MRI reads and report incidental findings, if any.

Re. @xlecours's second comment, do you mean how the module is accessible? from where?
if so; I would say:
This module can be directly accessed from the Loris Imaging Menu/Imaging Browser submenu, or indirectly linked from multiple other modules such as the Imaging Uploader (Number of MincInserted column), the DICOM archive (View Images in the MRI Browser column), or instrument list (View Imaging Data button under the Behavioral Battery of Instruments list).

Reply:
added the other types of users